inventory-api/src/main/scala/com/normation/inventory/domain/DataTypes.scala

```diff
@@ -93,6 +112,32 @@ final case class PublicKey(value : String) extends HashcodeCaching { assert(!isE
 
 }
 
+final case class Certificate(value : String) extends SecurityToken with HashcodeCaching { assert(!isEmpty(value))
+
+  // Value of the key may be stored (with old fusion inventory version) as one line and without rsa header and footer, we should add them if missing and format the key
+  val key = {
+    if (value.startsWith("-----BEGIN CERTIFICATE-----")) {
+      value
+    } else {
+      s"""-----BEGIN CERTIFICATE-----
+      |${value.grouped(80).mkString("\n")}
```

Comment:
Are we really in PEM serialisation format? I think x509 is "-----BEGIN x509 CERTIFICATE-----" but that must be validated